bindata/network/ovn-kubernetes/self-hosted/single-zone-interconnect/004-config.yaml

```diff
@@ -0,0 +1,87 @@
+---
```

Comment:
**Note**: the only difference between this file and the one in multi-zone in self-hosted is that it lacks the configuration for libovsdb logs.